src/js/utils/DataUtils.js

```diff
@@ -118,21 +103,19 @@ export const parseData = ({ data, properties: props }) => {
 
   let formattedData = [];
 
-  if (props.type === 'string' && props.format === 'date') {
+  if (props.type === 'string' && props.format === 'date')
     formattedData = parseDateData(data);
-  } else if (props.type === 'numerical') {
+  else if (props.type === 'numerical')
     formattedData = parseNumericalData(data, leftTaper, rightTaper, binSize);
-  } else {
-    formattedData = parseDataGeneral(data);
-  }
+  else formattedData = parseDataGeneral(data);
 
   // move 'missing' object to end of list
-  formattedData = binMissingData(formattedData);
+  const missingIndex = formattedData.findIndex((e) => e.x === 'missing');
+  if (missingIndex !== -1)
+    formattedData.push(formattedData.splice(missingIndex, 1)[0]);
 
   // order categories according to config file, where applicable
-  if (props.enum) {
-    formattedData = orderCategories(formattedData, props.enum);
-  }
+  if (props.enum) formattedData = orderCategories(formattedData, props.enum);
```

Comment:
Maybe this should be moved before the "missing to the end" part? I am afraid we are not guaranteed tiah the "missing" category will be at the end??